packages/gallery/tests/e2e/styleParams/overlayDesign.e2e.spec.js

```diff
@@ -0,0 +1,48 @@
+import { GALLERY_CONSTS } from 'pro-gallery-lib';
+import GalleryDriver from '../../drivers/pptrDriver';
+import { toMatchImageSnapshot } from '../../drivers/matchers';
+
+expect.extend({ toMatchImageSnapshot });
+
+describe('overlayDesign - e2e', () => {
+  let driver;
+
+  beforeAll(async () => {
+    driver = new GalleryDriver();
+    await driver.openPage();
+  });
+
+  afterAll(async () => {
+    await driver.closePage();
+  });
+
+  it('should set overlaySize to 50 percent, position-centered horizontally with padding 30', async () => {
+    await driver.navigate({
+      galleryLayout: GALLERY_CONSTS.layout.GRID,
+      overlayPosition: GALLERY_CONSTS.overlayPositions.CENTERED_HORIZONTALLY,
+      overlaySize: 50,
+      overlaySizeType: 'PERCENT',
+      overlayPadding: 30,
+      hoveringBehaviour: GALLERY_CONSTS.infoBehaviourOnHover.NO_CHANGE,
+    });
+    await driver.waitFor.hookToBeVisible('item-container');
+    await driver.waitFor.timer(200);
+    const page = await driver.grab.elemScreenshot('.pro-gallery');
+    expect(page).toMatchImageSnapshot();
```

Comment:
This code is duplicated :-( 
Makes it really hard to understand what is the difference between the tests.